src/test/java/net/imglib2/img/cell/CopyTest.java

```diff
@@ -130,6 +131,10 @@ public void copyWithIterationBoth( final Img< IntType > srcImg, final Img< IntTy
 		final Cursor< IntType > dst = dstImg.cursor();
 		while ( src.hasNext() )
 		{
+			if ( !dst.hasNext() )
+			{
+				throw new NoSuchElementException("Cursor does not have next element");
```

Comment:
Would it not be better to call JUnit's `fail` method here, for consistency? These methods, while public, are only intended for use from unit tests, right?